build-container/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM archlinux:base-devel
+
+# The following is from https://github.com/archzfs/archzfs-ci/blob/master/worker/Dockerfile
+RUN pacman -Syu --noconfirm --needed python-pipx python-twisted python-future git wget systemd-sysvcompat openresolv vi
```

Comment:
It would be nice to get rid of some of these dependencies at some point, but let's not create unnecessary work now. Excellent job with those comments, BTW! The are really helpful.